lib/block-supports/behaviors.php

```diff
@@ -99,24 +107,54 @@ function gutenberg_render_behaviors_support_lightbox( $block_content, $block ) {
 	$w->set_attribute( 'data-wp-interactive', true );
 	$w->set_attribute(
 		'data-wp-context',
-		sprintf( '{ "core":{ "image": { "initialized": false, "imageSrc": "%s", "lightboxEnabled": false, "lightboxAnimation": "%s", "hideAnimationEnabled": false } } }', $img_src, $lightbox_animation )
+		sprintf(
+			'{ "core":
+				{ "image":
+					{ 	"initialized": false,
+						"lightboxEnabled": false,
+						"hideAnimationEnabled": false,
+						"preloadInitialized": false,
+						"lightboxAnimation": "%s",
+						"imageSrc": "%s",
+						"imageSrcSet": "%s",
+						"targetWidth": "%s",
+						"targetHeight": "%s"
+					}
+				}
+			}',
+			$lightbox_animation,
+			$img_src,
+			$img_srcset ? $img_srcset : '',
```

Comment:
Had to change this, as null coalescing operator (??) is not present in PHP version 5.6 😢

Reply:
Great, thanks!